dotnet/docs/EXPERIMENTS.md

```diff
@@ -57,7 +58,7 @@ You can use the following diagnostic IDs to ignore warnings or errors for a part
 - SKEXP0052: Memory plugins
 - SKEXP0053: Microsoft 365 plugins
 - SKEXP0054: Web plugins
-- SKEXP0055: Text chunkcer plugin
+- SKEXP0055: Text chunker plugin
```

Comment:
Pre-existing, but "Text chunker" isn't a plugin

Reply:
not related after push